### ISSUE TITLE
Add Eritrea holidays

### ISSUE THREAD

**Issue by arkid15r** [vacanza/holidays#1179]
Links:
  - https://en.wikipedia.org/wiki/Public_holidays_in_Eritrea
  - https://www.timeanddate.com/holidays/eritrea/

**Comment by Prateekshit73** [comment]
We have four holidays that do not have a corresponding Tigrinya name: [Birth of the Prophet](https://en.wikipedia.org/wiki/Birth_of_the_Prophet), [May Day](https://en.wikipedia.org/wiki/May_Day), [Eid al-Fitr](https://en.wikipedia.org/wiki/Eid_al-Fitr), [Eid al-Adha](https://en.wikipedia.org/wiki/Eid_al-Adha).
Should we set Tigrinya as the default language, or not?

**Comment by KJhellico** [comment]
If we don't find a good source, we won't include this language, as usual.

**Comment by prateekshit-v** [comment]
According to this [website](http://www.eritrea.be/old/eritrea-religions.htm):

Core Holidays Consistently Present Every Year:

    New Year (Jan 1)

    Leddet (Geez Christmas) (Jan 7) 🇪🇷 Orthodox

    Timket (Epiphany) (Jan 19 or 20) 🇪🇷 Orthodox

    Tensae (Easter) (Movable date) 🇪🇷 Orthodox

    Festival of Mariam Dearit (Keren) (May 29)

    Keddus Yohannes (Geez New Year) (Sept 11) 🇪🇷 Orthodox

    Meskel (Finding of the True Cross) (Sept 27) 🇪🇷 Orthodox

    Eid al-Fitr (Movable, by moon)

    Eid al-Adha (Movable, by moon)

    Muharram (Islamic New Year) (Movable, by moon)

    Mawlid an-Nabi (Mohammed's Birthday) (Movable, by moon)

    Christmas (Western) (Dec 25)




NEW Holidays after [2007](https://web.archive.org/web/20081010180144/http://www.eritrea.be/old/eritrea-religions.htm) (but before [2011](https://web.archive.org/web/20110903130335/http://www.eritrea.be/old/eritrea-religions.htm)):

    Mariam Debre Sina (June 28)

    Debre Bizen Abune Libanos (Aug 11)


Now, Holidays included in the above list, but NOT on [Wikipedia](https://en.wikipedia.org/wiki/Public_holidays_in_Eritrea):

    Festival of Mariam Dearit (Keren)

    Mariam Debre Sina

    Debre Bizen Abune Libanos

    Keddus Yohannes (Geez New Year)

    Meskel (Finding of the True Cross)

    Muharram (Islamic New Year)

    Christmas (Dec 25/Gregorian)

Included in Wikipedia Public Holidays, but NOT in above list:

    International Women's Day (8 March)

    International Workers’ Day (1 May)

    Independence Day (24 May)

    Martyrs’ Day (20 June)

    Revolution Day (1 September)


Should we rely on this [website](http://www.eritrea.be/old/eritrea-religions.htm) or [Wikipedia](https://en.wikipedia.org/wiki/Public_holidays_in_Eritrea) ?

**Comment by KJhellico** [comment]
Religious holidays != public holidays. It would be good to find some official sources.

**Comment by prateekshit-v** [comment]
I've looked extensively for official sources to confirm the list of public holidays in Eritrea. Unfortunately, I haven't been able to find any definitive government publication or documentation listing national holidays. I checked the official Eritrean Ministry of Information website (https://shabait.com/), but it doesn’t appear to maintain a consistent or official archive of annual public holidays.

Most of the information currently available seems to be either from secondary sources (e.g., Wikipedia, timeanddate.com) or from community-maintained sites like `eritrea.be`, which list a mix of public and religious observances without clear government attribution.

**Comment by PPsyrius** [comment]
Yeah, I don't think I can find any Eritrean official-sounding sources as well (even for their 3 main banks website, one of them wasn't updated since like 2013)

Let's go with this:
- https://explore-eritrea.com/working-hours-and-holidays/
- http://www.eritrea.be/old/eritrea-religions.htm
- https://shabait.com/ -> individual holidays annoucements
- https://en.wikipedia.org/wiki/Public_holidays_in_Eritrea
- https://www.timeanddate.com/calendar/?year=2025&country=168 (cross-checking)
